### PR TITLE
esb: fix CRC setup configuration

### PR DESCRIPTION
**Pull request by valeriosetti**
Set the CRC's register properly (16 bits, 8 bits, none)

### CODE REVIEW

**Review by grochu — COMMENTED** [review]
Thanks for this PR, the changes look good. Just please add an entry in https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases/release-notes-changelog.rst covering the fix.

**Review by grochu — APPROVED** [review]

**Review by lemrey — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -75,6 +75,11 @@ Zigbee
 * Updated :ref:`Zigbee shell library <lib_zigbee_shell>`.
   For details, see `Libraries for Zigbee`_.
 
+ESB
+---
+
+* Fix: modified the "update_radio_crc()" function in order to correctly configure the CRC's registers (8 bits, 16 bits or none).
```
Comment (FrancescoSer) [inline doc/nrf/releases/release-notes-changelog.rst:81]:
```suggestion
* Fixed the ``update_radio_crc()`` function in order to correctly configure the CRC's registers (8 bits, 16 bits, or none).
```

**Review by FrancescoSer — CHANGES_REQUESTED** [review]
Hi! Just a small suggestion

**Review by FrancescoSer — APPROVED** [review]

**Comment by valeriosetti** [comment]
Hi @carlescufi, is there something missing on my side which prevents this pull request from being accepted?
Thanks :)

**Comment by lemrey** [comment]
Hi @valeriosetti , can you please sign the CLA and do a rebase? We can get this merged if CI becomes green.

**Comment by lemrey** [comment]
@valeriosetti thanks for you patience

**Pull request merged**